python/cudf/cudf/core/internals/where.py

```diff
@@ -0,0 +1,375 @@
+import warnings
+
+import numpy as np
+import pandas as pd
+
+import cudf
+
+
+def _normalize_scalars(col, other):
```

Comment:
I think type annotations would be useful here - ditto below

Reply:
Added type annotations 👍